### PR TITLE
Aerospike Server update

### PR DESCRIPTION
**Pull request by shriharirathod**
- Bumped the version of our aero server from 3.14.1.5 to
      4.2.0.10
    - Configured log rotation.

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

### CODE REVIEW

File: aerospike.logrotate.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/var/log/aerospike/aerospike.log {
+    su root root
+    rotate 5
+    dateext
+    size 50M
+    compress
+    olddir /var/log/aerospike/
+    postrotate
+        /bin/kill -HUP `pidof asd`
```
Comment (mahaveerdarade) [inline aerospike.logrotate.txt:9]: please see why asd needs to be restarted? can we logrotate without restarting asd?

File: aerospike.template.conf [hunk 2]
```diff
@@ -33,16 +33,22 @@ network {
 	}
```
Comment (mahaveerdarade) [inline aerospike.template.conf:34]: i m assuming hyc configurations (dirty, clean nm etc) will be present in this file.

Comment (shriharirathod) [reply aerospike.template.conf:34]: Yes they are.

File: aerospike.logrotate.txt (hunk 1 above)
Comment (shriharirathod) [reply aerospike.logrotate.txt:9]: The above config is directly used from aerospike documentation.

Kill -HUP will not kill the asd process itself, but is required to refresh the handle to log file.

**Review by mahaveerdarade — APPROVED** [review]
looks good.

**Pull request merged**